api/server/agent.go

```diff
@@ -32,21 +36,22 @@ func (tapi ToDDApi) Agent(w http.ResponseWriter, r *http.Request) {
 		// Make sure UUID string actually contains something
 		if len(uuid[0]) > 0 {
 			// Let's get the full list so we can identify the right agent if the user specified a short
-			full_agent_list := tdb.DatabasePackage.GetAgents()
+			full_agent_list, _ := tapi.tdb.GetAgents()
```

Comment:
Yup, I'll add error handling to all the places where I previously ignored the error. (I don't normally ignore errors, I think I had some reason for ignoring them in this PR, but it's better to just add them in.)